tests/migration_test.go

```diff
@@ -344,6 +344,84 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 
 			})
 		})
+		FContext("with auto converge enabled", func() {
+			var options metav1.GetOptions
+			var cfgMap *k8sv1.ConfigMap
+			var originalMigrationConfig string
+			var kubevirtConfig = "kubevirt-config"
+			BeforeEach(func() {
+				tests.BeforeTestCleanup()
+
+				// set unsafe migration flag
```

Comment:
copy paste error. should be "set autoconverge" in comment instead of "set unsafe migration"